CONTRIBUTING.md

```diff
@@ -327,6 +327,31 @@ def function_name(parameter1, parameter2):
 	# Function implementation
 ```
 
+Bulleted lists in docstrings do not throw an error if they are incorrectly formatted. Publish the doc locally to 
```

Comment:
I believe the correct term here is build since publish suggests uploading to the internet
```suggestion
Bulleted lists in docstrings do not throw an error if they are incorrectly formatted. Build the doc locally to 
```